client/components/purchase-detail/style.scss

```diff
@@ -152,3 +142,18 @@
 	float: left;
 	height: 14px;
 }
+
+.purchase-detail__button {
+	&[href] {
+		text-align: center;
+	}
+
+	&:not(.clipboard-button) {
```

Comment:
In what cases do we see this? If we need to account for this case I would suggest the inverse:
```
.purchase-detail__button {
     width: 80%;
}
.purchase-detail__button.clipboard-button {
    width: auto;
}
```